nav2_msgs/msg/WaypointStatus.msg

```diff
@@ -0,0 +1,5 @@
+# Waypoint state for robot in NavigateThroughPoses or WaypointFollower
+uint8 PENDING=0 # Waypoint is not processed
+uint8 COMPLETED=1 # Waypoint is completed
+uint8 SKIPPED=2 # Waypoint is skipped
+uint8 waypoint_status
```

Comment:
It might be useful to add in the posestamped here as well of the goal for context